### ISSUE TITLE
Composer package requires min-stability: dev

### ISSUE THREAD

**Issue by beporter** [onedesign/oneimgix#6]
This package can not be included in a project if the consuming project has its own `minimum-stability` set to something other than `dev`.

The issue is caused by this package's composer.json specifying a branch name for the `imgix-php` dependency instead of a target version:

```
  "imgix/imgix-php": "dev-master"  // (or `dev-main`)
```

Reproduction steps:

```shell
$ mkdir oneimgix-test

$ cd oneimgix-test

$ composer init --no-interaction

$ composer require onedesign/oneimgix
Using version ^1.2 for onedesign/oneimgix
./composer.json has been updated
Loading composer repositories with package information
Updating dependencies (including require-dev)
Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - Installation request for onedesign/oneimgix ^1.2 -> satisfiable by onedesign/oneimgix[1.2.0].
    - onedesign/oneimgix 1.2.0 requires imgix/imgix-php dev-master -> no matching package found.
```

To resolve this, the composer.json file should specify a target version following the guidelines composer uses to choose stability: https://getcomposer.org/doc/04-schema.md#package-links

**Comment by brianjhanson** [comment]
Hey @beporter! Sorry this has been lingering for so long, I completely missed it. 

I've just released 1.2.1 to address this issue. I'm guessing it's long past useful, but it's still there if you need it. 

Thanks!

**Comment by beporter** [comment]
No worries! If it helps even one other person in the future it's worth it. Thanks! 👍
